jme3-core/src/main/java/com/jme3/material/TechniqueDef.java

```diff
@@ -824,4 +824,4 @@ public TechniqueDef clone() throws CloneNotSupportedException {
 
         return clone;
     }
-}
+}
```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR.